### DISCUSSION TITLE
Not added to contributors list

### DISCUSSION THREAD

**Discussion by theyasub** [CommunityToolkit/Maui#603] [General]
Hello, my name is Botirali, I am a .NET developer in Uzbekistan.

I really like the new MAUI and I want to use it. I'm going to contribute your projects.I've created some changes and PR. Reviewer has approved it.May I join the ranks of contributors.Sorry if I said wrong.

**Comment by theyasub** [comment]
![image](https://user-images.githubusercontent.com/105649270/188459800-4b80393e-f26c-4bdc-ba45-63d348a0cfa7.png)
this is contribute list

**Comment by IeuanWalker** [comment]
@Rashidovicc you become a contributor once a PR is merged

**Reply by theyasub** [reply]
Ok I understand

**Comment by theyasub** [comment]
@VladislavAntonyuk I've created some changes and PR. Reviewer has approved it and merged. I want to contribute. We have a .NET community in Uzbekistan, and we want to contribute to such .NET open source projects. Can I contribute?

**Reply by VladislavAntonyuk** [reply]
You can, but contributions should bring some value to the product. You can choose some bugs or features from existing issues,

**Reply by TheCodeTraveler** [reply]
Hi @Rashidovicc! We have quite a few items labeled https://github.com/CommunityToolkit/Maui/labels/help%20wanted that we'd love your assistance with! 

`https://github.com/CommunityToolkit/Maui/labels/help%20wanted`

**Comment by CliffAgius** [comment]
Closed as answered
